scripts/travis_test.sh

```diff
@@ -1,13 +1,94 @@
 #!/bin/bash
 RV=0
-cd examples/linux
+
+# Run all examples; this assumes PWD is examples/script
+run_examples() {
+    # concolic assumes presence of ../linux/simpleassert
+    echo "Running concolic.py..."
+    HW=../linux/helloworld
+    SA=../linux/simpleassert
+    END_OF_MAIN=$(objdump -d $SA|awk -v RS= '/^[[:xdigit:]].*<main>/'|grep ret|tr  -d ' ' | awk -F: '{print "0x" $1}')
+    python ./concolic.py $END_OF_MAIN
+    if [ $? -ne 0 ]; then
+        return 1
+    fi
+
+    echo "Running count_instructions.py..."
+    python ./count_instructions.py $HW |grep -q Executed
```

Comment:
if `count_instructions.py` crashed, but managed to output `Executed` before it crashed, the exit status would still be 0 since it's grep's exit status, not `count_instructions.py` 's. since the main reason for doing this is to verify all the scripts work we should make sure to detect failures of the scripts themselves :) . i think one way to easily implement the fail fast would be adding `set -e` at the top which activates bash's fail fast mode

Reply:
actually i was wrong, i don't think just adding `set -e` will fix it

Reply:
+1 for `set -e`, but printing `Executed` is the very last thing `count_instructions.py` does, so if `Executed` ever made it to stdout, it'd be during a successful cleanup of when it flushes its buffer.

Reply:
ah good point. but technically `m.context['count']` could crash, and `Executed` would still be printed

https://github.com/trailofbits/manticore/blob/7be258605649111b1f457744c3a6e2343b6d63da/examples/script/count_instructions.py#L30

```
[I] mark ubuntu /m/h/c/manticore (dev-test-examples) ❯ python -c "print 3, 3/0, 3" | grep  3; echo $status
3
Traceback (most recent call last):
  File "<string>", line 1, in <module>
ZeroDivisionError: integer division or modulo by zero
0

```

Reply:
that is a super nitpick though, i'm fine with leaving this line as is. i'm fine with merging this pr once we do the fail fast thing!